models/modelArticles.js

```diff
@@ -27,7 +27,7 @@ function retrieveArticleDataById(article_id) {
 }
 
 function retrieveAllArticleData(topic, sort_by = "created_at", order = "DESC") {
-  const validTopics = ["mitch", "cats"];
+  const validTopics = ["mitch", "cats", "paper"];
```

Comment:
Hmmmm. This is one way to do it, sure, but on a live site, there would be FAR more topics that would need to be included in this list. What if a new topic were added? Someone would have to come into this file every single time and update the list. If only we had an endpoint already that could fetch all of the topics from our database*....

Reply:
*we do have that